packages/scaffold-config/src/frameworks.ts

```diff
@@ -149,7 +149,7 @@ export const WIZARD_FRAMEWORKS = [
       ])
     },
     codeGenFramework: 'react',
-    glob: '*.{js,jsx,tsx}',
+    glob: '*.{js,jsx,ts,tsx}',
```

Comment:
I am not sure if this makes sense - can you write React in a `ts` file? TSX doesn't work there - I guess you could non-JSX React :thinking:

Reply:
yeah you wouldn't be able to write any JSX but theoretically you could write a component that returns a string or returns `null`. Although, react-docgen doesn't recognize those as components by default anyway 🤔 

yeah I'm thinking I should revert this change

Reply:
reverted these changes in https://github.com/cypress-io/cypress/pull/24982/commits/d24360867c0febf7ef3aec96328f65a5bad61813